substrafl_examples/get_started/plot_substrafl_sklearn_fedavg.py

```diff
@@ -0,0 +1,523 @@
+"""
+=======================================
+Substrafl Skearn FedAvg on IRIS dataset
+=======================================
+
+This example illustrate an advanced usage of Substrafl, and proposes a model training by federated learning
+using the federated averaging strategy.
+
+We will implement a custom algorithm to show how to interface SubstraFL with Sklearn.
+
+This example is based on:
```

Comment:
missing space? The bullet list is messed up on RTD
<img width="730" alt="image" src="https://user-images.githubusercontent.com/5354301/200014167-c5e0e9c1-6299-4f39-8a09-9d4cf56a40d7.png">